packages/react-core/src/components/TimePicker/TimePicker.tsx

```diff
@@ -221,7 +226,27 @@ export class TimePicker extends React.Component<TimePickerProps, TimePickerState
   };
 
   scrollToIndex = (index: number) => {
-    this.getOptions()[index].offsetParent.scrollTop = this.getOptions()[index].offsetTop;
+    // fixes issue where inline value for menutAppendTo results in initial menu item that is scrolled to being out of view
```

Comment:
I think explaining what is being done to fix the issue would help on top of calling out what issue is being fixed.